### PR TITLE
feat: Add cUSDC and aUSDC on mainnet

### LINKED ISSUE #10: List Compound and Aave tokens in the default token list
## What is the feature about

It is common for on-chain treasuries to park their $USDC holdings into a lending protocol such as Compound or Aave.

## Feature request

Include Compound and Aave tokens in the "Default tokens" list. For example, the cUSDC v3 token on Mainnet:

https://etherscan.io/token/0xc3d688B66703497DAA19211EEdff47f25384cdc3

Our "Assets" page does not show our Compound deposit at the moment:

SCR-20240403-nfhx

### PR DESCRIPTION
**Pull request by jmealy**
Resolves #10

### CODE REVIEW

File: src/tokens/safe.tokenlist.json [hunk 1]
```diff
@@ -71,6 +71,22 @@
...
       "chainId": 100,
       "logoURI": "https://assets.coingecko.com/coins/images/31099/standard/OLAS-token.png"
+    },
+    {
+      "symbol": "cUSDCv3",
+      "name": "Compound USDC",
+      "address": "0xc3d688b66703497daa19211eedff47f25384cdc3",
+      "decimals": 18,
+      "chainId": 1,
+      "logoURI": "https://safe-transaction-assets.safe.global/tokens/logos/0xc3d688B66703497DAA19211EEdff47f25384cdc3.png"
```
Comment (jmealy) [inline src/tokens/safe.tokenlist.json:81]: Not sure if this logo exists. Do I need to add it somewhere?

Comment (katspaugh) [reply src/tokens/safe.tokenlist.json:81]: We can ask tx-service to add it.

**Review by katspaugh — APPROVED** [review]
👍

File: src/tokens/safe.tokenlist.json [hunk 2]
```diff
@@ -71,6 +71,22 @@
       "decimals": 18,
       "chainId": 100,
       "logoURI": "https://assets.coingecko.com/coins/images/31099/standard/OLAS-token.png"
```
Comment (katspaugh) [inline src/tokens/safe.tokenlist.json]: Could you aso change this URL to safe-transaction-assets.safe.global?

**Pull request merged**